CMakeLists.txt

```diff
@@ -136,7 +136,7 @@ if(NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE Valgrind)
 endif(NOT CMAKE_BUILD_TYPE)
 
-set(CMAKE_CXX_FLAGS_DEBUG "-g")
+set(CMAKE_CXX_FLAGS_DEBUG "-O0 -g")
```

Comment:
How about `-0g`? I forget if clang supports that, but g++ definitely does, and it helps keep down the bloat a bit.

Reply:
Thanks, that will be better. Clang also has it since 4.0.